frontend/src/routes/BillabilityMonthlyWorkingHours.jsx

```diff
@@ -111,18 +129,33 @@ const BillabilityMonthlyWorkingHours = () => {
     dataIndex: "year",
     key: "year",
     render: (title) => <b>{title}</b>,
+    align: "right",
   });
   for (let i = 1; i < 13; i++) {
     columns.push({
       title: i,
       dataIndex: i,
       key: i,
+      render: (value) => {
```

Comment:
You could create a helper function that you import and use in all the places where you want to convert a number to locale string. That way it's easier to make modifications to it later on and also don't need to copy anything else than the function name to each of the render properties of table columns that need conversion to locale string.